frontends/web/src/routes/account/walletconnect/dashboard.tsx

```diff
@@ -34,29 +35,30 @@ type TProps = {
 }
 
 export const DashboardWalletConnect = ({ code, accounts }: TProps) => {
-  const isInitialized = useInitialization();
   const { t } = useTranslation();
+  const { web3wallet, isWalletInitialized } = useContext(WCWeb3WalletContext);
   const [sessions, setSessions] = useState<SessionTypes.Struct[]>();
   const receiveAddresses = useLoad(getReceiveAddressList(code));
 
-  const updateSessions = () => {
-    setSessions(Object.values(web3wallet?.getActiveSessions()));
-  };
+  const updateSessions = useCallback(() => {
+    const activeSessions = Object.values(web3wallet?.getActiveSessions());
```

Comment:
in the POC it does not use getActiveSessions but passes the session do remove as argument and filters the existing sessions. Does this work the same way?